src/jdk.jpackage/macosx/classes/jdk/jpackage/internal/MacBaseInstallerBundler.java

```diff
@@ -134,6 +136,20 @@ protected void validateAppImageAndBundeler(
                         I18N.getString(
                             "message.app-image-requires-app-name.advice"));
             }
+            if (Optional.ofNullable(
+                    SIGN_BUNDLE.fetchFrom(params)).orElse(Boolean.FALSE)) {
+                // if signing bundle with app-image, warn user if app-image
+                // is not allready signed.
```

Comment:
nitpicking: typo "allready" -> "already"

Reply:
will fix this spelling error